models/transfers/polygon/erc20/transfers_polygon_erc20.sql

```diff
@@ -0,0 +1,109 @@
+{{ config(
+    alias='erc20',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['transfer_type', 'evt_tx_hash', 'evt_index', 'wallet_address'],
+    post_hook='{{ expose_spells(\'["polygon"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["soispoke", "dot2dotseurat", "tschubotz", "hosuke"]\') }}'
+    )
+}}
+
+with sent_transfers as (
+    select 'send'           as transfer_type,
+           evt_tx_hash,
+           evt_index,
+           to             as wallet_address,
+           contract_address as token_address,
+           evt_block_time,
+           value            as amount_raw
+    from
+        {{ source('erc20_polygon', 'evt_transfer') }}
+    {% if is_incremental() %}
+    where evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+),
+received_transfers as (
+    select 'receive'                          as transfer_type,
+           evt_tx_hash,
+           evt_index,
+           "from"                             as wallet_address,
```

Comment:
I am not sure for now we need to use dune sql syntax for spellbook, as the dbt-test still uses spark sql syntax to check, and `"from"` may cause dbt-test failure at the moment.

Reply:
I used it since @jeff-dude asked to make the query future proof (i.e. compatible with Dune SQL) already [here](https://github.com/duneanalytics/spellbook/pull/2953#issuecomment-1476527976).

Reply:
yes -- sorry @Hosuke for not tagging, but we'll try to push views to be compatible on both engines where possible (not always at this time).